alfalfa_worker/model_advancer_base.py

```diff
@@ -105,16 +104,16 @@ def run(self):
         self.set_idle_state()
         self.init_sim()
         self.set_db_status_running()
-        self.ac.redis_pubsub.subscribe(self.site_id)
+        self.redis_pubsub.subscribe(self.site_id)
```

Comment:
notice that we no longer have to say `self.ac` or `self.model_logger.logger`. So much nicer!